build.rtc/src/glad.c

```diff
@@ -852,15 +852,19 @@ static PFNWGLGETPROCADDRESSPROC_PRIVATE gladGetProcAddressPtr;
 
 static
 int open_gl(void) {
-#ifndef IS_UWP
+// begin post-generator edit
+#if defined(RTC_WINDOWS_UNIVERSAL)
+    libGL = LoadPackagedLibrary(L"opengl32.dll",0);
+#else
     libGL = LoadLibraryW(L"opengl32.dll");
+#endif // defined(RTC_WINDOWS_UNIVERSAL)
+// end post-generator edit
```

Comment:
This looks like a change we should push upstream, such that we won't need to carry it any further.

Reply:
I did mention it here https://github.com/Dav1dde/glad/issues/141. 

I'm not sure why the UWP function isn't used in place upstream but can ask, maybe it would be OK now.

Reply:
My guess is that upstream they don't need/want the `RTC_WINDOWS_UNIVERSAL` tag, and would want to use their own tag.